tests/Resources/Page/ODH/ODHDashboard/ODHDashboard.robot

```diff
@@ -375,6 +375,49 @@ Search Items In Resources Section
     Sleep   5
     Input Text  xpath://input[@class="pf-c-search-input__text-input"]       ${element}
 
+Open Element By Name
+    [Documentation]     It opens the element by enabling Quick Start filter in Resources Section of ODS Dashboard.
+    [Arguments]     ${element}
+    Click Link      Resources
+    Sleep   5
+    Select Checkbox     //input[@id="quickstart--check-box"]
+    Wait Until Page Contains Element    //article[@id="${element}"]
+    Click Link          //article[@id="${element}"]//a
+    Wait Until Page Contains ELement    //div[@class="pf-c-drawer__panel-main"]     5
+
+Get The Count Of Description
+    [Documentation]     Gives the count of description which is shown while starting the tour.
+    ${count}=   Get ELement Count   //div[@class="pfext-quick-start-content"]/ul/li
+    Click Button    //button[@data-test="Start button"]
+    Wait Until Page Contains Element    //div[@class="pfext-quick-start-content"]
+    [Return]    ${count}
+
+Verify The Progress Of The Items
+    [Documentation]     Verifies the label on the item card which shows the progress and closes the sidebar.
+    [Arguments]     ${element}  ${progress_string}
+    Wait Until Page Contains Element    //button[@data-testid="qs-drawer-close"]    5
+    Click Button    //button[@data-testid="qs-drawer-close"]
+    Page Should Not Contain Element     //div[@class="pf-c-drawer__panel-main"]
+    IF  '${progress_string}'=='In Progress'
+        Element Text Should Be      //article[@id="${element}"]//span[@class="pf-c-label pf-m-purple pf-m-outline"]     In Progress
+    ELSE IF     '${progress_string}'=='Complete'
+        Element Text Should Be      //article[@id="${element}"]//span[@class="pf-c-label pf-m-green pf-m-outline"]      Complete
+    END
+
+Move To Next Description By Clicking Yes
+    Wait Until Page Contains Element    //input[@data-testid="qs-drawer-check-yes"]
+    Click Button    //input[@data-testid="qs-drawer-check-yes"]
+    Click Button    //button[@data-testid="qs-drawer-next"]
+
+Move To Next Description By Clicking No
+    Wait Until Page Contains Element    //input[@data-testid="qs-drawer-check-no"]
+    Click Button    //input[@data-testid="qs-drawer-check-no"]
+    Click Button    //button[@data-testid="qs-drawer-next"]
+
+Verify Previous Description
+    Click Button    //button[@data-testid="qs-drawer-back"]
+    Page Should Contain Element     //div[@class="pfext-quick-start-task-header"]
```

Comment:
I think it would be a better check to verify that the new step the user reached after clickin on "back", is actually the expected one

Reply:
Okay I'll work on it.